src/udev/udev-manager.c

```diff
@@ -477,29 +473,46 @@ static void worker_attach_event(Worker *worker, Event *event) {
         event->worker = worker;
 
         (void) event_reset_time_relative(
-                        e,
-                        &event->timeout_warning_event,
-                        CLOCK_MONOTONIC,
+                        manager->event,
+                        &worker->timeout_warning_event_source,
+                        CLOCK_BOOTTIME,
```

Comment:
Hmmm? commit says no effective change yet you're changing the clock of the timer here. Feels worth mentioning in the commit message or maybe a separate commit with justification.

Reply:
The change is dropped, and now CLOCK_MONOTONIC used again. I thought CLOCK_BOOTTIME was better, but that's a bad idea here.